README.md

```diff
@@ -39,4 +39,16 @@ If changes you make require migrations, or you merge migrations made by others i
 ```console
 > user@machine:~/dev-directory$ docker compose exec -it app bash
 > root@container-id:/virtual-instrument-museum/vim-app$ python manage.py migrate
+```
+
+## Loading data
+
+At present, VIM supports an initial list of 262 instruments curated from Wikidata, and their names in English and French. These instruments can be found in `web-app/django/startup_data/vim_instruments_with_images-15Sept.csv`.
+
+Two django management commands are provided to load these initial instruments. These should be run from within the app Docker container.
+
+```bash
+> user@machine:~/dev-directory$ docker compose exec -it app bash
+> root@container-id:/virtual-instrument-museum/vim-app$ python manage.py import_languages
```

Comment:
As noted elsewhere: as it currently stands, `import_languages` doesn't actually import anything - it creates two objects with values that are hard-coded. Guessing now that it's named the way it is so that it matches `import_instruments`. If this is the case, perhaps add a comment in `import_languages` noting that the current code within it is a placeholder, and that the command will eventually actually import things.